DOCS-3002: Add instructions for physically connecting components to how-tos (#3535)
viamrobotics/docs: docs/how-tos/image-data.md

{{% expand "A configured camera. Click to see instructions." %}}

> First, make sure to connect your camera to your machine if it's not already connected (like with an inbuilt webcam on a Macbook).

Reviewer: ```suggestion
First, make sure to connect your camera to your machine if it's not already connected (like with an inbuilt laptop webcam).
```
